### PR TITLE
Add SVG Race

### PR DESCRIPTION
**Pull request by kirjs**
I wrote this last year and wouldn't want to lose lol 
image

### CODE REVIEW

File: apps/kirjs/src/app/modules/svg-race/player/player.component.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+@Component({
+  // tslint:disable-next-line:component-selector
+  selector: '[kirjs-player]',
+  templateUrl: './player.component.html',
+  styleUrls: ['./player.component.css']
+})
+export class PlayerComponent implements OnInit, AfterViewInit, OnChanges {
+  @ViewChild('guess', {static: true}) guess: ElementRef;
+  @Input() path;
+  @Input() d = '';
```
Comment (aziz512) [inline apps/kirjs/src/app/modules/svg-race/player/player.component.ts]: use a better name?

File: apps/kirjs/src/app/modules/svg-race/timer/timer.component.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'kirjs-timer',
+  templateUrl: './timer.component.html',
+  styleUrls: ['./timer.component.css']
+})
+export class TimerComponent implements OnInit {
```
Comment (aziz512) [inline apps/kirjs/src/app/modules/svg-race/timer/timer.component.ts]: This component isn't used anywhere

File: libs/utils/src/lib/sync/components/poll/sync-poll.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'slides-poll',
+  templateUrl: './sync-poll.component.html',
+  styleUrls: ['./sync-poll.component.css']
+})
+export class SyncPollComponent implements OnInit {
```
Comment (aziz512) [inline libs/utils/src/lib/sync/components/poll/sync-poll.component.ts]: Unused component

File: libs/utils/src/lib/sync/directives/all-viewer-values.directive.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  ngAfterViewInit() {
+
+    this.sync.whenPresenting$.subscribe(() => {
+      this.sync.getAllViewersValues(this.syncAllUserValues).subscribe(values => {
+        this.values = Object.entries(values).map(([key, value]) => ({key, value}));
+      });
+    });
+    // this.sync.whenViewing$
```
Comment (aziz512) [inline libs/utils/src/lib/sync/directives/all-viewer-values.directive.ts]: clean up the comments

File: libs/utils/src/lib/sync/directives/sync-viewer-value.directive.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        this.control.valueAccessor.writeValue(value);
+        console.log('sub here');
+        return this.control.valueChanges;
+      })).subscribe((value) => {
+        console.log('VALUE');
+        this.sync.updateViewerValue(this.syncViewerValue, value);
+      });
+    });
+
+    // this.sync.whenViewing$
```
Comment (aziz512) [inline libs/utils/src/lib/sync/directives/sync-viewer-value.directive.ts]: clean up the comments

File: libs/utils/src/lib/sync/sync.component.ts [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'codelab-sync',
+  templateUrl: './sync.component.html',
+  styleUrls: ['./sync.component.css']
+})
+export class SyncComponent implements OnInit {
```
Comment (aziz512) [inline libs/utils/src/lib/sync/sync.component.ts]: unused component

File: apps/kirjs/src/app/modules/svg-race/player/player.component.ts (hunk 1 above)
Comment (NothingEverHappens) [reply apps/kirjs/src/app/modules/svg-race/player/player.component.ts]: Wanna keep it consistent with the attribute name. Blame svg lol

File: libs/utils/src/lib/sync/components/poll/sync-poll.component.ts (hunk 3 above)
Comment (NothingEverHappens) [reply libs/utils/src/lib/sync/components/poll/sync-poll.component.ts]: This is coming soon, I just created so someone can add shit there

File: apps/kirjs/src/app/modules/svg-race/timer/timer.component.ts (hunk 2 above)
Comment (NothingEverHappens) [reply apps/kirjs/src/app/modules/svg-race/timer/timer.component.ts]: Going to be used it the game, though it should prob be made generic

**Pull request merged**